doc/concepts/data_model/migrations.rst

```diff
@@ -113,14 +113,14 @@ This method is described in the README file of the
     There are also two other methods that we **do not recommend**,
     but you may find them useful for one reason or another.
 
-    **Method 3**: the ``tarantoolctl`` utility
+    **Method 3**: the :ref:`tt <tt-cli>` utility
 
-    The ``tarantoolctl`` utility ships with Tarantool.
-    Connect to the necessary instance using ``tarantoolctl connect``.
+    The ``tt`` utility ships with Tarantool.
```

Comment:
As I know, it does not. There is a separate package for `tt`, which is a recommended dependency of the `tarantool` package.

Reply:
Fixed.